src/tools/rbd_mirror/PoolReplayer.cc

```diff
@@ -311,9 +288,9 @@ void PoolReplayer<I>::init()
   auto cct = reinterpret_cast<CephContext *>(m_local_io_ctx.cct());
   librbd::api::Config<I>::apply_pool_overrides(m_local_io_ctx, &cct->_conf);
 
-  std::string local_mirror_uuid;
+  // XXXMG: should we care about mirror_uuid is being changed later?
```

Comment:
Not sure if mirror_uuid change is handled correctly in the current master, but I suppose here it is broken, because I read it only once on PoolReplayer creation. Probably can be revised and improved later.